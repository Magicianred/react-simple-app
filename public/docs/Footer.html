<!DOCTYPE html>
<html lang="en">
<head>
    
    <meta charset="utf-8">
    <title>Footer - Documentation</title>
    
    
    <script src="scripts/prettify/prettify.js"></script>
    <script src="scripts/prettify/lang-css.js"></script>
    <!--[if lt IE 9]>
      <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
    <link type="text/css" rel="stylesheet" href="styles/prettify.css">
    <link type="text/css" rel="stylesheet" href="styles/jsdoc.css">
    <script src="scripts/nav.js" defer></script>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
</head>
<body>

<input type="checkbox" id="nav-trigger" class="nav-trigger" />
<label for="nav-trigger" class="navicon-button x">
  <div class="navicon"></div>
</label>

<label for="nav-trigger" class="overlay"></label>

<nav >
    
    <h2><a href="index.html">Home</a></h2><h3>Classes</h3><ul><li><a href="Contact.html">Contact</a></li><li><a href="Footer.html">Footer</a></li><li><a href="Header.html">Header</a></li><li><a href="Home.html">Home</a></li><li><a href="NotFound.html">NotFound</a></li><li><a href="WhoAre.html">WhoAre</a></li></ul>
</nav>

<div id="main">
    
    <h1 class="page-title">Footer</h1>
    

    




<section>

<header>
    
        <h2>
        Footer
        </h2>
        
    
</header>

<article>
    
        <div class="container-overview">
        
            

    

    <h4 class="name" id="Footer"><span class="type-signature"></span>new Footer<span class="signature">()</span><span class="type-signature"></span></h4>

    




<dl class="details">

    
    <dt class="tag-source">Source:</dt>
    <dd class="tag-source"><ul class="dummy"><li>
        <a href="Partials_Footer.js.html">Partials/Footer.js</a>, <a href="Partials_Footer.js.html#line11">line 11</a>
    </li></ul></dd>
    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    
</dl>





<div class="description usertext">
    <p>Component for showing the footer of the site</p>
</div>









    <h5>Example</h5>
    
    <pre class="prettyprint"><code>return (
 &lt;Footer />
)</code></pre>





















        
        </div>
    

    

    

    
    
    

     

    

    

    

    

    
</article>

</section>




    
    
</div>

<br class="clear">

<footer>
    Documentation generated by <a href="https://github.com/jsdoc3/jsdoc">JSDoc 3.6.3</a> on Tue Mar 17 2020 20:23:12 GMT+0100 (Central European Standard Time) using the <a href="https://github.com/clenemt/docdash">docdash</a> theme.
</footer>

<script>prettyPrint();</script>
<script src="scripts/polyfill.js"></script>
<script src="scripts/linenumber.js"></script>



</body>
</html>