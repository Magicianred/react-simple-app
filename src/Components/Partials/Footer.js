import React from 'react';

/**
 * Functional Component for showing the footer of the site
 * @component
 * @example
 * return (
 *  <Footer />
 * )
 */
const Footer = () => {
    return (
        <footer className="partial footer">
            <p>
                This is the footer 
                - <a
                className="App-link"
                href="https://reactjs.org"
                target="_blank"
                rel="noopener noreferrer"
                >
                Learn React
                </a>
            </p>
        </footer>
    )
}

export default Footer;