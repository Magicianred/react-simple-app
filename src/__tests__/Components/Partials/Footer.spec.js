import React from 'react';
import { create } from 'react-test-renderer';

import Footer from './../../../Components/Partials/Footer'

describe("Footer Partials Component", () => {

    describe("Snapshots", () => {
    
        it("Matches the snapshot (with default title)", () => {
            const instance = create(<Footer />);
            expect(instance.toJSON()).toMatchSnapshot();
        });
    
    });

    
});