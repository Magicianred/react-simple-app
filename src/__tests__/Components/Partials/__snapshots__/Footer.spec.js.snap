// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Show the Footer site Snapshots Matches the snapshot (with default params) 1`] = `
<footer
  className="partial footer"
>
  <p>
    This is the footer - 
    <a
      className="App-link"
      href="https://reactjs.org"
      rel="noopener noreferrer"
      target="_blank"
    >
      Learn React
    </a>
  </p>
</footer>
`;
